src/shared/components/common/pictrs-image.tsx

```diff
@@ -39,7 +39,7 @@ export class PictrsImage extends Component<PictrsImageProps, any> {
             "img-expanded slight-radius":
               !this.props.thumbnail && !this.props.icon,
             "img-blur": this.props.thumbnail && this.props.nsfw,
-            "rounded-circle img-cover img-icon me-2": this.props.icon,
```

Comment:
There`s an `me-` without a number at the end of the classnames.

Reply:
It's a `1`; the color contrast might have gotten you here